pyiron_contrib/workflow/has_to_dict.py

```diff
@@ -10,6 +12,8 @@ def to_dict(self):
     def _repr_json_(self):
         return self.to_dict()
 
-    @property
     def info(self):
         print(dumps(self.to_dict(), indent=2))
+
+    def repr_json(self):
```

Comment:
There shoud be a __repr_json__ or similar method used in notebooks to Display the object automatically as json